data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/actions/TailSamplerAggregateActionConfig.java

```diff
@@ -6,20 +6,24 @@
 package org.opensearch.dataprepper.plugins.processor.aggregate.actions;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.AssertTrue;
 
 import java.time.Duration;
 
 public class TailSamplerAggregateActionConfig {
+    @JsonPropertyDescription("period to wait before considering that a trace event is complete")
     @JsonProperty("wait_period")
     @NotNull
     private Duration waitPeriod;
     
+    @JsonPropertyDescription("percent value to use for sampling non error events.")
     @JsonProperty("percent")
     @NotNull
     private Integer percent;
     
+    @JsonPropertyDescription("condition that determines if an event is error event or not")
```

Comment:
Should link to the expression documentation and be clear if this is a Data Prepper conditional expression